### PR TITLE
API server now must be started before controller-manager. Fix this in Kubemark startup script

### PR DESCRIPTION
**Pull request by gmarek**

### CODE REVIEW

File: test/kubemark/start-kubemark-master.sh [hunk 1]
```diff
@@ -50,4 +48,8 @@ kubernetes/server/bin/kube-apiserver \
 	--secure-port=443 \
 	--basic-auth-file=/srv/kubernetes/basic_auth.csv &> /var/log/kube-apiserver.log &
 
+# kube-contoller-manager now needs running kube-api server to actually start
+sleep 5
```
Comment (wojtek-t) [inline test/kubemark/start-kubemark-master.sh]: Can we make it in more robust way - e.g. wait until apiserver is responding to /healthz or sth like that?

Comment (gmarek) [inline test/kubemark/start-kubemark-master.sh]: Done. @wojtek-t PTAL

Comment (wojtek-t) [inline test/kubemark/start-kubemark-master.sh]: Thanks - LGTM

**Comment by wojtek-t** [comment]
LGTM

**Pull request merged**